src/System.Memory/tests/Binary/ReverseEndiannessUnitTests.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Buffers.Binary.Tests
+{
+    public class ReverseEndianessUnitTests
+    {
+        [Fact]
+        public void ReverseEndianness_Byte_DoesNothing()
```

Comment:
nit: `ReverseEndianness_ByteAndSByte_DoesNothing` to follow the naming scheme of other tests here